### PR TITLE
fix: supply both image index and image manifest digests when creating…

### PR DESCRIPTION
**Pull request by ralphbean**
… pyxis image record

See also https://github.com/redhat-appstudio/release-service-utils/pull/134

### CODE REVIEW

File: tasks/create-pyxis-image/create-pyxis-image.yaml [hunk 1]
```diff
@@ -126,11 +126,16 @@ spec:
...
             MEDIA_TYPE=$(skopeo inspect --raw "docker://${PULLSPEC}" | jq -r .mediaType)
-            ARCHS=($(get-image-architectures "${PULLSPEC}"))
-            for (( a=0; a < ${#ARCHS[@]}; a++)); do
-                ARCH=${ARCHS[$a]}
-                skopeo inspect --no-tags --override-os linux --override-arch "$ARCH" "docker://${PULLSPEC}" \
+            ARCH_BASE64_DETAILS=($(get-image-architectures "${PULLSPEC}"))
+            for (( a=0; a < ${#ARCH_BASE64_DETAILS[@]}; a++)); do
+                ARCH_DETAIL=$(echo ${ARCH_BASE64_DETAILS[$a]} | base64 -d)
+                OS=$(jq '.platform.os' <<< "$ARCH_DETAIL")
+                ARCH=$(jq '.platform.os' <<< "$ARCH_DETAIL")
```
Comment (scoheb) [inline tasks/create-pyxis-image/create-pyxis-image.yaml]:
```suggestion
                ARCH=$(jq '.platform.architecture' <<< "$ARCH_DETAIL")
```

**Review by johnbieren — CHANGES_REQUESTED** [review]
This can't be merged without the task images updated (which I know you can't do until the utils PR is merged, but marking this here so it is not accidentally merged)

File: tasks/create-pyxis-image/README.md [hunk 2]
```diff
@@ -20,6 +20,9 @@ by a new line.
 | snapshotPath | Path to the JSON string of the mapped Snapshot spec in the data workspace | Yes | mapped_snapshot.json |
 | dataPath | Path to the JSON string of the merged data to use in the data workspace. Only required if commonTags is not set or empty. | Yes | data.json |
 
+## Changes in 2.2.1
```
Comment (johnbieren) [inline tasks/create-pyxis-image/README.md]: Should be 2.2.2

Comment (ralphbean) [reply tasks/create-pyxis-image/README.md]: Updated!

**Review by johnbieren — COMMENTED** [review]
Code changes LGTM, but not approving yet as we need https://github.com/konflux-ci/release-service-utils/pull/134 to merge and then the image used for these three tasks to be updated to the resulting image

**Comment by johnbieren** [comment]
> This can't be merged without the task images updated (which I know you can't do until the utils PR is merged, but marking this here so it is not accidentally merged)

The image tag to use for these tasks is `b3059c67d4bdb6c9cf1739fdfb2c0cb595fe0f86`

**Comment by ralphbean** [comment]
> The image tag to use for these tasks is b3059c67d4bdb6c9cf1739fdfb2c0cb595fe0f86

Updated.

**Review by johnbieren — APPROVED** [review]

**Pull request merged**